scorched/connection.py

```diff
@@ -286,9 +282,7 @@ def _prepare_docs(self, docs):
                 # fields
                 if value is None:
                     continue
-                if name in self._datefields:
```

Comment:
True. But this check was moved, not removed. See below first lines of is_datetime_field (lines 136, 137)